### ISSUE TITLE
Feat: Ignore specific unlisted dependencies for Ant Design

### ISSUE THREAD

**Issue by mathieucaroff** [webpro/knip#383]
Me and my team use the Ant Design component library and from time to time, we need to import file types which come from the internal component definitions. The thing is, the ant-design library is structured in a unique way, with an npm package declared for each base component. These base components follow a naming convention: `rc-<component-name>` (rc stands for React Component). So we end up with this output in the `Unlisted dependencies` section of the knip report:

```
rc-picker/lib/interface         src/components/form/field/DateRangePicker/DateRangePickerField.tsx
rc-picker/lib/panels/TimePanel  src/components/form/field/DateRangePicker/DateRangePickerField.tsx
rc-tree-select/lib/TreeSelect   src/components/form/field/TreeSelectField/TreeSelectField-types.ts
rc-tree-select/lib/interface    src/components/form/field/TreeSelectField/TreeSelectField-types.ts
rc-select/lib/BaseSelect        src/routes/home/targets/_shared/Tags/form/_components/TagsCreateModal/TagsCreateModal-types.ts
rc-select/lib/BaseSelect        src/routes/home/targets/devices/form/tagsToDevicesAdd/TagsToDevicesAddForm-data.ts
rc-picker/lib/interface         src/routes/home/targets/discovery/onboarding/devices/_utils/table.tsx
rc-picker/lib/interface         src/routes/home/targets/discoveryOld/onboarding/devices/_utils/table.tsx
```

I don't want us to have to add a dependency to the `package.json` each time we depend on a new `rc-*` component for types, therefore it would be nice if we could exclude the `^rc-.*` package name pattern from the "unlisted dependencies" section.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.6.0. See [Release 3.6.0](https://github.com/webpro/knip/releases/tag/3.6.0) for release notes._

**Comment by webpro** [comment]
Thanks for raising this, should help with in other situations too. Added to docs: https://knip.dev/reference/configuration/#ignoredependencies
